mne/tests/test_annotations.py

```diff
@@ -794,4 +795,76 @@ def test_read_annotation_txt_orig_time(
     assert_array_equal(annot.description, ['AA', 'BB'])
 
 
+def test_annotations_simple_iteration():
+    """Test indexing Annotations."""
+    NUM_ANNOT = 5
+    EXPECTED_ELEMENTS_TYPE = (np.float64, np.float64, np.str_)
+    EXPECTED_ONSETS = EXPECTED_DURATIONS = [x for x in range(NUM_ANNOT)]
+    EXPECTED_DESCS = [x.__repr__() for x in range(NUM_ANNOT)]
+
+    annot = Annotations(onset=EXPECTED_ONSETS,
+                        duration=EXPECTED_DURATIONS,
+                        description=EXPECTED_DESCS,
+                        orig_time=None)
+
+    for ii, elements in enumerate(annot[:2]):
+        assert isinstance(elements, OrderedDict)
+        expected_values = (ii, ii, str(ii))
+        for elem, expected_type, expected_value in zip(elements.values(),
+                                                       EXPECTED_ELEMENTS_TYPE,
+                                                       expected_values):
+            assert np.isscalar(elem)
+            assert type(elem) == expected_type
+            assert elem == expected_value
+
+    # unpacking is not allowed
```

Comment:
What does this mean? Unpacking like this doesn't work because there are 4 values to unpack and you only supply 3 names.

Reply:
you cannot untuple a dict. I made it bit more explicit.

Reply:
But you can unpack a dict:

```
d = dict(a=1, b=2, c=3)
x, y, z = d
```

Which is probably not what the user wants though, because this returns the keys. If you want the values, you can change that to:

```
x, y, z = d.values()
```

Reply:
yes!!, you are actually right. The test makes no sense.

Reply:
I didn't find a way to bloc the usage. I guess that [we are all consenting adults](https://python-guide-chinese.readthedocs.io/zh_CN/latest/writing/style.html#we-are-all-consenting-adults)